src/SortMovies/SortMovies.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component, useState } from 'react';
+import moment from 'moment';
+import './SortMovies.scss';
+
+export class SortMovies extends Component {
+    constructor(props) {
+        super(props)
+
+        this.state = {
+            value: ''
+        }
+    }
+
+    updateSortValue = (event) => {
+        this.setState({ [event.target.name]: event.target.value})
+    }
+
+    getOptionValue = () => {
+        this.props.sortMovies(this.state.value)
+    }
+
+    render() {
+        return (
+            <div className="sort-container">
+              <h1 className="cards-title">All movies</h1>
+              <div className="sort-control">
+                <h4 className="sort-title">sort by</h4>
+
+                <select
```

Comment:
You'll almost definitely want to use `userEvent` for testing this dropdown. Look into their `selectOptions` method.